tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -9,3 +9,19 @@ spec:
     SESSION_TIMEOUT = 3600
 status:
   readyCount: 1
+---
+# Test the status code is correct for each endpoint
+# This test is for heat endpoints
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      set -x
+      RETURN_CODE=0
+      PUBLIC_URL=$(oc get route -n $NAMESPACE horizon-public -o jsonpath='{.status.ingress[0].host}')
+      STATUSCODE=$(curl --silent --output /dev/stderr --head --write-out "%{http_code}" $PUBLIC_URL/healthcheck)
```

Comment:
Do we really have this path available ? This is provided by the healthcheck middleware used by OpenStack services but I don't think we have it for horizon.

Reply:
Ah true. I copied from another operator. I'll fix this.